dags/glam.py

```diff
@@ -325,8 +324,7 @@
                 project_id=billing_project_id,
                 date_partition_parameter=None,
                 arguments=("--replace",),
-                sql_file_path=extracts_sql_path,
-                parameters=(f"channel:STRING:{channel}",),
+                sql_file_path=f"sql/moz-fx-data-shared-prod/{dataset_id}/glam_client_probe_counts_{channel}_extract/query.sql",
```

Comment:
Should in be this instead?
```suggestion
                sql_file_path=f"sql/moz-fx-data-shared-prod/{dataset_id}/glam_extract_firefox_{channel}_v1/query.sql",
```